lib/teiserver/coordinator/consul_commands.ex

```diff
@@ -1510,6 +1510,21 @@ defmodule Teiserver.Coordinator.ConsulCommands do
     ConsulServer.say_command(cmd, state)
   end
 
+  def handle_command(%{command: "set-config-teaser", remaining: new_teaser} = cmd, state) do
+    ConsulServer.say_command(cmd, state)
+    new_teaser = String.trim(new_teaser)
+
+    new_teaser =
+      case Regex.run(~r/^[a-zA-Z0-9_\-\[\] \<\>\+\|:]+$/, new_teaser) do
```

Comment:
hum, what?
What is this thing supposed to do? At the very least a comment explaining that would be welcome.
I see that you copied that from the `rename` command right? Perhaps putting that in a module-private function would help clarifying the intent behind this regex.